src/Microsoft.Azure.WebJobs.Host/Filters/FunctionInvocationFilterAttribute.cs

```diff
@@ -5,12 +5,13 @@
 using System.Threading;
 using System.Threading.Tasks;
 
-namespace Microsoft.Azure.WebJobs.Host
+namespace Microsoft.Azure.WebJobs.Host.Filters
 {
     /// <summary>
     /// Base class for declarative function invocation filters.
     /// </summary>
-    public abstract class InvocationFilterAttribute : Attribute, IFunctionInvocationFilter
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = true, Inherited = true)]
+    public abstract class FunctionInvocationFilterAttribute : Attribute, IFunctionInvocationFilter
```

Comment:
Renaming so attribute and interface names are better in sync. Other filters follow the same pattern